releases/0.2.0.ramstage.md

```diff
@@ -0,0 +1,223 @@
+   > `WARNING ! This will modify your HOST flash chip content !`. When the
+   > `Enter "yes" to confirm:` prompt appears, type `yes` and press enter.
+
+   At the end of the process (it may take several minutes) you should have
+   something like this:
+
+   ```
+   About to program "/tmp/flash.pnor" at 0x00000000..0x04000000 !
+   Programming & Verifying...
+   [==================================================] 100% ETA:0s
+   ```
+
+4. * Log into the BMC GUI at https://<BMC_IP>/. \
+     Make sure to use `https`.
+   * Enter the Server power operations
+     `https://<BMC_IP>/#/server-control/power-operations` and invoke
+     warm reboot.
+   * Then move to Serial over LAN remote console
+     `https://<BMC_IP>/#/server-control/remote-console` to observe
+     whether the platform is booting.
+
+#### Buidling coreboot image
+
+In order to build coreboot image, follow the steps below:
+
+1. Clone the coreboot repository:
+
+   ```
+   git clone git@github.com:3mdeb/coreboot.git -b TBD_TAG_NAME
+   # or HTTPS alternatively
+   git clone https://github.com/3mdeb/coreboot.git -b TBD_TAG_NAME
+   ```
+   `talos_2_support_ramstage` - ramstage devlopment branch - merge requests should go here.\
+   `squashed_talos_2_support` - upstream branch, can be regularly pushed with force.\
+   `talos_2_support` - legacy branch for bootblock and romstage release - as of today nothing should be pushed here.
+
+2. Get the submodules:
+
+   ```
+   cd coreboot
+   git submodule update --init --checkout
+   ```
+
+3. Start docker container (assuming you are already in coreboot root
+   directory):
+
+   ```
+   docker run --rm -it -v $PWD:/home/coreboot/coreboot -w /home/coreboot/coreboot docker.pkg.github.com/3mdeb/coreboot/coreboot-sdk /bin/bash
+   ```
+
+4. When inside of the container, configure the build for Talos II:
+
+   ```
+   make menuconfig
+   ```
+
+   * Navigate to the **Mainboard** submenu.
+   * As a **Mainboard vendor** select `Raptor Computing Systems`.
+   * Select Exit and then navigate to the **Payload** submenu.
+   * In **Add a payload** select `skiboot`.
+   * Save the configuration and exit.
+
+   ![make menuconfig mainboard](../images/cb_menuconfig_ramstage_mainboard.png)
+   ![make menuconfig payload](../images/cb_menuconfig_ramstage_payload.png)
+
+
+5. Start the build process of coreboot inside the container:
+
+   ```
+   make
+   ```
+
+#### Running the coreboot on Talos II
+
+1. At the end of build process you should see `Built raptor-cs/talos-2 (Talos II)`.
+   Copy the result binary from `<coreboot_dir>/build/coreboot.rom.signed.ecc` to the BMC
+   (assuming in the coreboot root directory):
+
+   ```
+   scp build/bootblock.signed.ecc root@<BMC_IP>:/tmp/bootblock.signed.ecc
+   scp build/coreboot.rom.signed.ecc root@<BMC_IP>:/tmp/coreboot.rom.signed.ecc
+   ```
+
+2. Backup the HBB partition (for faster later recovery) by invoking this
+   command on BMC:
+
+   ```
+   pflash -P HBB -r /tmp/hbb.bin
```

Comment:
If we want to keep this step we should also make a backup of HBI, but I don't know if this is even necessary, flashing whole PNOR is also an option.